packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -90,21 +90,23 @@ const deploymentGroup = codedeploy.ServerDeploymentGroup.import(this, 'ExistingC
 You can [specify a load balancer](https://docs.aws.amazon.com/codedeploy/latest/userguide/integrations-aws-elastic-load-balancing.html)
 with the `loadBalancer` property when creating a Deployment Group.
 
+`LoadBalancer` is an abstract class with static factory methods that allow you to create instances of it from various sources.
+
 With Classic Elastic Load Balancer, you provide it directly:
 
 ```ts
 import lb = require('@aws-cdk/aws-elasticloadbalancing');
 
 const elb = new lb.LoadBalancer(this, 'ELB', {
-    // ...
+  // ...
 });
 elb.addTarget(/* ... */);
 elb.addListener({
-    // ...
+  // ...
 });
 
 const deploymentGroup = new codedeploy.ServerDeploymentGroup(this, 'DeploymentGroup', {
-    loadBalancer: elb,
+  loadBalancer: codedeploy.LoadBalancer(elb),
```

Comment:
Can this code sample be correct?

Reply:
I was just testing if you're paying attention. You've passed.